src/Dapr.Actors.AspNetCore/ActorsServiceCollectionExtensions.cs

```diff
@@ -52,6 +52,7 @@ public static void AddActors(this IServiceCollection services, Action<ActorRunti
                     {
                         JsonSerializerOptions = options.JsonSerializerOptions,
                         DaprApiToken = options.DaprApiToken,
+                        HttpEndpoint = options.HttpEndpoint,
```

Comment:
One of the impactful changes I had to make here is making the HTTP endpoint used by the actor client configurable. This was not configurable outside of the env-var.